packages/auth-client/src/controllers/engine.ts

```diff
@@ -172,7 +172,10 @@ export class AuthEngine extends IAuthEngine {
       h: {
         t: "eip4361",
       },
-      p: pendingRequest.cacaoPayload,
+      p: {
+        ...pendingRequest.cacaoPayload,
+        iss,
+      },
       s: respondParams.signature,
     };
```

Comment:
I don't see where the [spec](https://github.com/WalletConnect/walletconnect-docs/pull/355/files) mentions how we want to return the address/`iss`. But is this the right place for it? the `h`,`p` and `s` are all part of a specific CAIP spec. I might be mistaken and would appreciate a correction

Reply:
The return of the `p` remains the same. That's why I'm merging whatever `iss` is with the `cacao payload`

Reply:
to clarify, before `iss` was already included in the `cacao payload` by default. Now we have access to it at later point

Reply:
Right! slipped my mind